components/gitpod-db/go/encryption.go

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License.AGPL.txt in the project root for license information.
+
+package db
+
+import (
+	"bytes"
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/rand"
+	"encoding/base64"
+	"errors"
+	"fmt"
+)
+
```

Comment:
Maybe [this](https://github.com/gitpod-io/gitpod/pull/15360/files#r1050471510) is worth mentioning here at the top as well.